pkgs/development/python-modules/pulumi-hcloud/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  parver,
+  pulumi,
+  pythonOlder,
+  semver,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pulumi-hcloud";
+  version = "1.20.4";
+
+  pyproject = true;
+  build-system = [ setuptools ];
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "pulumi";
+    repo = "pulumi-hcloud";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-m9MRXDTSC0K1raoH9gKPuxdwvUEnZ/ulp32xlY1Hsdo=";
+  };
+
+  sourceRoot = "${src.name}/sdk/python";
+
+  propagatedBuildInputs = [
+    parver
+    pulumi
+    semver
+  ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests? Could you leave a comment in the code?

Reply:
I copied this from a [similar package](https://github.com/NixOS/nixpkgs/blob/ea35048e5d734c549cff91baa0af8fcc51318885/pkgs/development/python-modules/pulumi-aws/default.nix#L37-L38) but it seems tests works there (I was wrongly assuming tests were similar). I've just removed that line. Thanks :)

Reply:
The tests don't run at the moment.
Could you add the pytestCheckHook to the nativeCheckInputs or explicitly disable the test and leave a comment?

> Make sure the tests are enabled using for example [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) and, in the case of libraries, are passing for all interpreters. If certain tests fail they can be disabled individually. Try to avoid disabling the tests altogether. In any case, when you disable tests, leave a comment explaining why.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Thanks!